cmd/kcp-front-proxy/options/options.go

```diff
@@ -43,6 +46,11 @@ func NewOptions() *Options {
 func (o *Options) AddFlags(fss *cliflag.NamedFlagSets) {
 	o.Proxy.AddFlags(fss)
 	logsapiv1.AddFlags(o.Logs, fss.FlagSet("logging"))
+
+	// add flags that are filtered out from upstream, but overridden here with our own version
+	fss.FlagSet("KCP").Var(kcpfeatures.NewFlagValue(), "feature-gates", ""+
```

Comment:
This this should be default ? appended?

Reply:
How do you mean? My thought was that there were no FGs on the front-proxy yet, so I can just copy-paste this into it.